javascript/javascript1/week4/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
Very good. Nice to see you have used string methods like 'startsWith',endsWith and included extra commands

Reply:
thank you)